v2/bundle/jwtbundle/bundle.go

```diff
@@ -0,0 +1,62 @@
+package jwtbundle
+
+import (
+	"crypto"
+	"io"
+
+	"github.com/spiffe/go-spiffe/v2/spiffeid"
+)
+
+// Bundle is a collection of trusted JWT public keys for a trust domain.
+type Bundle struct {
+}
+
+// New creates a new bundle.
+func New(trustDomain spiffeid.TrustDomain) *Bundle {
+	panic("not implemented")
+}
+
+// Load loads a Bundle from a file on disk.
+func Load(trustDomain spiffeid.TrustDomain, path string) (*Bundle, error) {
+	panic("not implemented")
+}
+
+// Read decodes a bundle from a reader.
+func Read(trustDomain spiffeid.TrustDomain, r io.Reader) (*Bundle, error) {
+	panic("not implemented")
+}
+
+// Parse parses a bundle from bytes.
+func Parse(trustDomain spiffeid.TrustDomain, b []byte) (*Bundle, error) {
+	panic("not implemented")
+}
+
+// TrustDomain returns the trust domain the bundle belongs to.
+func (b *Bundle) TrustDomain() spiffeid.TrustDomain {
+	panic("not implemented")
+}
+
+// JWTKeys returns the JWT keys in the bundle.
+func (b *Bundle) JWTKeys() map[string]crypto.PublicKey {
```

Comment:
I missed some basic crud ops on these bundles (for adding/removing material). I'll add.